simulator/func_sim/func_sim.h

```diff
@@ -46,6 +46,13 @@ class FuncSim : public Simulator
             PC = target.address;
             sequence_id = target.sequence_id;
         }
+
+        /* GDB interfaces */
```

Comment:
I think new interfaces should not be sold as GDB interfaces, it's better to use common names.
Additionally, "do not repeat yourself" and use these functions in high-order functions. For example:
```c++
run( const std::string& tr, int steps)
{
    load( tr);
    run( steps);
}

Reply:
That is still actual. Using "gdb_" name is confusing, as it brings outer scope idioms to the inner scope.